edsnlp/pipelines/core/terminology/terminology.py

```diff
@@ -92,7 +92,7 @@ def __init__(
             ignore_excluded=ignore_excluded,
         )
 
-        self.phrase_matcher.build_patterns(nlp=nlp, terms=terms)
+        self.phrase_matcher.build_patterns(nlp=nlp, terms=terms, progress=True)
```

Comment:
So we only want to display the progress bar when using the `TerminologyMatcher` but not for any `build_patterns` of `SimstringMatcher` or `EDSPhraseMatcher`, correct? Could you briefly develop your idea?